test/extensions/filters/http/common/fuzz/uber_filter.h

```diff
@@ -54,6 +58,7 @@ class UberFilterFuzzer {
   Http::FilterFactoryCb cb_;
   NiceMock<Envoy::Network::MockConnection> connection_;
   Network::Address::InstanceConstSharedPtr addr_;
+  NiceMock<Envoy::StreamInfo::MockStreamInfo> stream_info_;
```

Comment:
non-blocking comment: if fuzzing this would give extra coverage, in the future this could be fuzzed as well

Reply:
Ack, let me look into this. I see there's already a function used to fuzz the stream info

Reply:
Well i'm having lifetime issues with `TestStreamInfo`, since it's not copiable but can throw an exception when creating. I'll look into in another PR.

For now, I kept the stream info as an arg for the function. This will allow ESPv2 fuzz tests to pass in a fuzzed one instead of the mocked one.

Reply:
If can throw while creating? 

Sure, as an arg is fine -- I think the future if it's fuzzed it may not need to be held as a member (or just held as a sharedptr) and constructed per iteration.

Reply:
Yes, this function sometimes throws exceptions: https://github.com/envoyproxy/envoy/blob/a78a6d1edd50be17f8fa0790ae27502580434977/test/fuzz/utility.h#L133

I don't have the exception logged anymore, but it has to do with parsing a network address.